sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -195,13 +201,37 @@ public String getLogs(@NonNull String group, @IntRange(from = 0) int limit, @Non
             ContentValues values = iterator.next();
             Long dbIdentifier = values.getAsLong(DatabaseManager.PRIMARY_KEY);
 
+            /*
+             * When we can't even read the identifier (in this case ContentValues is most likely empty).
+             * That probably means it contained a record larger than 5MB and we hit the cursor limit.
+             * Get rid of first non pending log.
+             */
+            if (dbIdentifier == null) {
+                AppCenterLog.error(LOG_TAG, "Empty database record, probably entry larger than 5MB, need to delete as it's now corrupted");
```

Comment:
This happens with 2.75 MB file for me, so I'm not sure about this value

Reply:
We can change the log message.

Reply:
"entry larger than .." -> "an entry is larger than .."